src/mod_ping.erl

```diff
@@ -64,7 +44,7 @@ route_ping_iq(JID, Server) ->
     Pid = self(),
     T0 = erlang:monotonic_time(millisecond),
     F = fun(_From, _To, Acc, timeout) ->
-               ejabberd_c2s_info_handler:call(Pid, mod_ping, timeout),
+               ejabberd_c2s:call_remote_hook(Pid, mod_ping, timeout),
```

Comment:
What's wrong with `run_hook`? `run` is already used with hooks.

Reply:
Let's think of greppability, too.

Reply:
my intention was to make it very explicit that this is not a regular hook call - this hook is not called in the context of the caller, like all the others. Greppability shouldn't be an issue, because eventually there is a call to `mongooseim_hooks`.